app/api/event_tag_endpoint.py

```diff
@@ -0,0 +1,38 @@
+from tracardi.service.storage.drivers.elastic.tag import add_tags, remove_tags, load_all_tags
+from pydantic import BaseModel
+from typing import List
+from fastapi import APIRouter
+from app.config import server
+from tracardi.domain.value_object.bulk_insert_result import BulkInsertResult
+
+
+class TagForm(BaseModel):
+    event_type: str = ""
+    tags: List[str] = []
+
+
+router = APIRouter()
+
+
+@router.post("/event/tag/add", tags=["event"], include_in_schema=server.expose_gui_api, response_model=dict)
+async def add_event_tag_endpoint(tag_form: TagForm):
+    updated, result = await add_tags(
```

Comment:
I understand that 1 return parameter is number of updated records. But add_tags has it hard coded. Elastic returns the number of updated records. Use this value.

Reply:
Refactor all driver functions to use number of updated records that returns elastic.